ci/build_container/build_recipes/zlib.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+VERSION=1.2.11
+
+wget -O zlib-$VERSION.tar.gz https://github.com/madler/zlib/archive/v$VERSION.tar.gz
+tar xf zlib-$VERSION.tar.gz
+cd zlib-$VERSION
+./configure --prefix=$THIRDPARTY_BUILD
+make V=1 install
```

Comment:
Hi @dnoe, thanks for the review and the instructions! I'm pushing a new commit shortly.